pkg/collector/hostname_collector.go

```diff
@@ -0,0 +1,119 @@
+package collector
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	//autoscalingv2 "k8s.io/api/autoscaling/v2beta2"
+	autoscalingv2 "k8s.io/api/autoscaling/v2"
+)
+
+const (
+	HostnameMetricType = "requests-per-second"
+	HostnameRPSQuery   = `scalar(sum(rate(%s{host=~"%s"}[1m])) * %.4f)`
+)
+
+type HostnameCollectorPlugin struct {
```

Comment:
Let's change `Hostname` to `RPS` in all these cases. Same with the filename `rps_collector.go`?

Reply:
I'm ok with it, just worried to people confusing it with skipper_collector RPS metric in the future. I know it is different, but for someone relatively new to the code it might be confusing, if you think this is ok I don't mind changing it.

Reply:
It's not a big deal to me. If you think it's easier to understand as it is, then that is ok from my perspective.

I know the code base well so I may be biased 🙃

Reply:
I think your idea is better, but I'm still worried with name confusion, what do you think about `HostnameRPSCollector` or `ExternalRPSCollector`, the second emphasising the fact this is the collector for the `External` type?

Reply:
`ExternalRPSCollector` I like that